extensions/adobe/experience/adcloud/partnerdata.schema.json

```diff
@@ -0,0 +1,37 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/adcloud/partnerdata",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adcloud partner data",
+  "type": "object",
+  "description": "Details about AdCloud partners.",
+  "definitions": {
+    "partnerdata": {
+      "properties": {
+        "xdm:id": {
```

Comment:
You cannot redefine the type of `"xdm:id"`. It is already defined as a string https://github.com/adobe/xdm/blob/master/schemas/context/identity.schema.json

Please convert this to a string or define a different property.

Reply:
converted to string